dotCMS/src/main/java/org/apache/felix/framework/OSGIUtil.java

```diff
@@ -141,12 +140,13 @@ private Properties defaultProperties() {
      * @param context The servlet context
      * @return Framework
      */
-    public Framework initializeFramework(ServletContext context) {
+    public synchronized Framework initializeFramework(ServletContext context) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use block level rather than method level synchronization](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572140)